index.js

```diff
@@ -45,7 +45,7 @@ const exportDoc = function(namespaces, format) {
   for (const ns of Object.keys(result.namespaces)) {
     const nsPath = path.join(basePath, ...ns.split('.'));
     const nsFilePath = path.join(nsPath, `index.${ext}`);
-    mkdirp.sync(nsFilePath.substring(0, nsFilePath.lastIndexOf('/')));
+    mkdirp.sync(nsFilePath.substring(0, nsFilePath.lastIndexOf(path.sep)));
```

Comment:
That’s what I said in my comment I thought. The other 3 places (lines 24, 62, and 68) where that lastIndexOf(‘/’) for path is done all use forward slash delimited paths. Only that one place ends up needed to be OS-specific.